.tav.yml

```diff
@@ -413,13 +413,14 @@ body-parser:
     - node test/sanitize-field-names/express.js
 
 aws-sdk:
-  # We want this version range:
+  # We want a version range something like this:
   #   versions: '>=2.858 <3'
   # However, awk-sdk releases *very* frequently (almost every day) and there
   # is no need to test *all* those releases. Instead we statically list every
   # N=5 releases to test.
   #
   # Maintenance note: This should be updated periodically.
-  versions: '2.858.0 || 2.863.0 || 2.868.0 || 2.873.0 || 2.878.0 || 2.883.0 || 2.888.0 || 2.893.0 || 2.898.0 || 2.903.0 || 2.908.0 || 2.913.0 || 2.918.0 || >2.918 <3'
+  versions: '2.876.0 || 2.881.0 || 2.886.0 || 2.891.0 || 2.896.0 || 2.901.0 || 2.906.0 || 2.911.0 || 2.916.0 || 2.921.0 || 2.926.0 || 2.931.0 || 2.936.0 || >2.918 <3'
```

Comment:
Thanks for updating. Do you think it worthwhile to maintain that initial minimum `2.858` version? Then perhaps jump up to a recent few versions.

Reply:
no strong opinion on whether it's worthwhile, but an easy enough change to make. Done.